_sass/messages.scss

```diff
@@ -12,11 +12,19 @@ $error: #BF433A;
   background: transparentize($info, 0.8);
   border-color: $info;
   color: $info;
+
+  p {
+    color: $info;
```

Comment:
Question: Is this actually a use case where `!important` on the above `color` definition is actually the correct solution

Reply:
I don't think so, as that would mean that links were blue (rather than orange) too, which we probably don't want.

Reply:
The more specific selector of `.doc p` means that `!important` does not effect the styling.
![image](https://user-images.githubusercontent.com/1496834/141596930-294b04f2-a132-4807-b490-0d6d6adef654.png)